src/contracts/v3-ava-caps-26-02-2023/v3-ava-caps-26-02-2023.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import '../common/StewardBase.sol';
+import {AaveV3Avalanche, AaveV3AvalancheAssets} from 'aave-address-book/AaveV3Avalanche.sol';
+
+/**
+ * @dev This steward sets configure reserve as collateral for Link.e and wAVAX on AAVE V3 Avalanche
+ * - Snapshot: todo 
+ * - Dicussion: https://governance.aave.com/t/arc-chaos-labs-supply-and-borrow-cap-updates-aave-v3-2023-02-24/12048
+ */
+contract AaveV3AvaCapsSteward is StewardBase {
+    
+    address public constant DAIe = AaveV3AvalancheAssets.DAIe_UNDERLYING;
```

Comment:
these assignments are a bit unnecessary, but perhaps taste

Reply:
Ok, good point. In my following PRs I will remove those assignments.